src/store/modules/nft.js

```diff
@@ -879,6 +885,8 @@ const actions = {
       signedMessageText: data.signedMessageText,
       enableSignatureImage: data.enableSignatureImage,
       isHidden: data.isHidden,
+      tableOfContents: data.tableOfContents,
+      blurb: data.blurb,
```

Comment:
Please don't add this yet, we would probably be using `https://schema.org/Review` in either iscn or listing data, but it is not there in book press and not confirmed yet